tools/BLASTp_Bootcamp.md

```diff
@@ -0,0 +1,115 @@
+# BlastP Bootcamp
```

Comment:
I really like the title, but can you please rename the markdown document to be consistent in formatting with the other tools. `BLASTp.md`